### PR TITLE
docker: Build the docker code with webpack

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

**Comment by dperpeet** [comment]
Is it worth compressing svg files? Do you have some numbers to support that?

**Comment by stefwalter** [comment]
Same answer as for HTML files:

```
[stef@falcon images]$ ls -l
total 12
-rw-rw-r--. 1 stef stef 5048 17. Aug 13:42 drive-harddisk-symbolic.svg
-rw-rw-r--. 1 stef stef 1855 17. Aug 13:42 drive-harddisk-symbolic.svg.gz
```

File: pkg/docker/containers.js [hunk 1]
```diff
@@ -17,19 +17,22 @@
...
+
+    var Mustache = require("mustache");
+    var service = require("service");
+
+    var client = require("./client");
+    var overview = require("./overview");
+    var container_details = require("./details");
+    var image_details = require("./image");
+    var storage = require("./storage.jsx");
+    require("./run");
```
Comment (larsu) [inline pkg/docker/containers.js]: Do we need this here? It wasn't in there before and `image.js` already pulls it.

**Comment by larsu** [comment]
The SVG compression is not actually used anywhere, is it? At least I don't see any .svg.gz in the tarball.

File: pkg/docker/containers.js (hunk 1 above)
Comment (stefwalter) [inline pkg/docker/containers.js]: Probably not. It may have been me trying to figure out a load ordering issue.

**Comment by stefwalter** [comment]
> The SVG compression is not actually used anywhere, is it? At least I don't see any .svg.gz in the tarball.

gz files aren't distributed. Check in the installed directory to see if it took.

**Comment by larsu** [comment]
> gz files aren't distributed. Check in the installed directory to see if it took.

Uh, of course. Sorry for the confusion.

Except for the `./run` issue, this looks good to merge.

File: pkg/docker/containers.js (hunk 1 above)
Comment (stefwalter) [inline pkg/docker/containers.js]: Removed.

**Pull request closed without merging**